src/plugins/add-cal/add-calendar-fr.hbs

```diff
@@ -0,0 +1,74 @@
+---
+{
+	"title": "Ajout au calendrier",
+	"language": "fr",
+	"category": "Plugiciels",
+	"description": "",
+	"tag": "add-cal",
+	"parentdir": "add-cal",
+	"altLangPrefix": "add-cal",
+	"dateModified": "2021-04-28"
+}
+---
+
+<h2>Événement incluant un endroit détaillé</h2>
+<p>L'exemple suivant présente un événement incluant un emplacement détaillé (nom, adresse, région, province et code postale).</p>
+<div class="wb-addcal provisional mrgn-bttm-md" typeof="Event">
+	<h3 property="name">Canux20</h3>
+	<p property="description">CanUX (prononcé «canucks», comme l&apos; équipe de hockey) est un événement indépendant, local, à but non lucratif et dirigé par des bénévoles, créé pour les personnes qui souhaitent apprendre et partager leur expertise sur l'expérience utilisateur et la façon dont notre profession contribue à faire les choses fonctionnent mieux pour tout le monde.</p>
+	<p property="location" typeof="Place">
+		<span property="name">Centre Shaw</span><br>
+		<span class="address" property="address" typeof="PostalAddress">
+			<span property="streetAddress">55, promenade du Colonel-By</span><br>
+			<span property="addressLocality">Ottawa</span>,
+			<span property="addressRegion">ON</span><br>
+			<span property="postalCode">K1N 9J2</span>
+		</span>
+	</p>
+	<dl>
+		<dt>Début</dt>
+		<dl><time property="startDate" datetime="2020-01-27 07:00Z">Le lundi 27 janvier 2020, 7:00am (HNE)</time></dl>
```

Comment:
`<dd>` instead of `<dl>`

Reply:
Changes have been made and submitted via commit amend.